app/editor/src/components/upload/Upload.tsx

```diff
@@ -16,6 +16,7 @@ export interface IUploadProps
   extends Omit<ButtonHTMLAttributes<HTMLButtonElement>, 'onSelect' | 'value'> {
   onSelect?: (e: React.ChangeEvent<HTMLInputElement>) => void;
   onDownload?: () => void;
+  onPostDelete?: () => void;
```

Comment:
Rename to `onDelete`, and rename the current function that handles the input to `handleDelete`

Reply:
Fixed it.